data/heroes/green/5star.yml

```diff
@@ -1333,3 +1333,32 @@
     - This character’s Troops deal 100% more additional damage against weaker element.
     - This character receives 10% less additional damage from Troops of stronger element.
   image: https://i.imgur.com/5xvOFq1.jpg
+- name: Ekanite
+  class: Fighter
+  source: Ninja
+  family: Ninja
+  speed: Charge
+  power: 841
+  attack: 828
+  defense: 780
+  health: 1491
+  skill: Order Ambush
+  effects:
+    - "Special Skill has a different effect when charged with 100%/200%/300% mana:"
+    - "x1 Mana Charge:"
+    - " - Deals 200% damage to all enemies."
+    - " - Summons a Shinobi Minion for each ally with 8% HP and 10% attack inherited from the caster."
+    - " - Shinobi Minion gives its owner the ability to counterattack with 60% of the damage received for as long as the owner has a Shinobi Minion. The effect can’t be dispelled."
+    - "x2 Mana Charge:"
+    - " - Deals 280% damage to all enemies."
+    - " - Summons a Shinobi Minion for each ally with 16% HP and 16% attack inherited from the caster."
+    - " - Shinobi Minion gives its owner the ability to counterattack with 60% of the damage received for as long as the owner has a Shinobi Minion. The effect can’t be dispelled."
+    - "x3 Mana Charge:"
+    - " - Deals 360% damage to all enemies."
+    - " - Summons a Shinobi Minion for each ally with 22% HP and 26% attack inherited from the caster."
+    - " - Shinobi Minion gives its owner the ability to counterattack with 60% of the damage received for as long as the owner has a Shinobi Minion. The effect can’t be dispelled."
+  types:
+    - Damage All
+    - Minion Summoner
+    - Counterattack
+  image: https://i.imgur.com/YQskc4O.jpg
```

Comment:
FYI - normally I'd colorize the background to match the green.  Going to approve anyways, but we should consider circling back and re-doing this one.